_posts/The importance_of_being_earnest.md

```diff
@@ -0,0 +1,262 @@
+justification, neither was there any proportional
+spacing between the letters, i.e. the letters ‘i’
+and ‘w’ would take the same amount of space.
+This increased the paper usage by almost two
+times. Moreover, pasting bits and pieces like the
+aforementioned ‘α’ added to the overall bulk of a
+document.
+
+The sheer human labour leaves us incredulous.
+The above begs the question, what did one do
+for graphs back then? Ravichandran is a trained
+mechanical draftsman; he would therefore take
+the rough scribbles students would bring and
+draw a neat, proportional graph by hand. Multiple
+copies would then then taken via Xeroxing.
+In such cases, he also used stencils in a technique
+called stencil cutting and duplicating to trace out
+alphabets onto the graph (they couldn’t be
+keyed in with a typewriter). Requisite “slots”
+(silhouettes) would be cut out; rolled up and ink
+passed through.
+
+He then says, with pride strengthening his voice, “I must have seen thousands of theses in my life so
+far, so intimately. I would even be acknowledged
+in the document. So many students have now
+become big professors and hold prominent
+administrative positions. Many of them even
+come back sometimes, make small talk. You see,
+they do not forget.”
+
+From when he was working well past midnight as
+a draftsman at Electrical Engineering or slaving
+over a student’s thesis to handling four machines
+single-handedly, Ravichandran’s single-minded
+dedication to his work is very apparent. It has
+even rubbed off on his assistants too; “the boys
+are so sincere that they are reluctant to take off
+even on festivals and holidays”.
+
+He then takes a trip down IIScian memory
+lane: “The IISc of thirty-five years ago was very
+different. There was only one tiny hostel for
+women then. Initially, we set up the Xerox Centre
+at Gymkhana. But then on popular demand”, he
+grins, “we moved to a more accessible place:
+Union Office, near Nesara. Then, twenty years
+ago, we moved to this location.”
+
+Like every baby boomer, he too takes a minute
+and goes on the “aaj kal ke bacche” rant. He
+speaks of the increasing impatience in Gen Y. He
+laments the loss of skill too: of penmanship and
+artistry- “especially as a draftsman, it is saddening
+to see this. We used so many instruments, for
+so many different types of drawings: isometric,
+orthogonal… It is all obsolete now, in a time when
+everything is increasingly being taken for granted”
+The wistfulness is still apparent as Ravichandran
+yet again says “we used to work together with the
+students. I literally and metaphorically had a
+hand in their theses. Four hours a day would be
+devoted to solely this. Students would often take
+me out for a stroll and some tea at the end of a
+day’s grueling work”. Even as he speaks, he is
+finished with binding another manuscript.
+“Well, this is how your Campus Xerox Centre has
+come to be. This is how I’ve spent my whole life:
+at IISc.”
+
```

Comment:
it seems that the text was just copied from some PDF and pasted directly here; need to preview the markdown's output to ensure that all line breaks are proper.

Reply:
Its fine on Markdown, it only has a line break in the output if you give two line spaces in the Markdown